src/main/java/com/manardenza/controller/HotelController.java

```diff
@@ -11,25 +13,35 @@
 public class HotelController {
 
     private HotelService hotelService;
+    private CurrentUser currentUser;
+    private org.slf4j.Logger log = LoggerFactory.getLogger(HotelController.class);
 
-    public HotelController(HotelService hotelService) {
+    public HotelController(HotelService hotelService, CurrentUser currentUser) {
         this.hotelService = hotelService;
+        this.currentUser = currentUser;
     }
 
     public List<Hotel> findHotelByName(String hotelName) {
+        log.info(String.format("User %s has searched for hotels by name %s", currentUser.getUser().getFullName(),
+            hotelName));
         return hotelService.findHotelByName(hotelName);
     }
 
     public List<Hotel> findHotelByCity(String cityName) {
+        log.info(String.format("User %s has searched for hotels by city %s", currentUser.getUser().getFullName(),
+            cityName));
         return hotelService.findHotelByCity(cityName);
     }
 
+    //TODO: consider deleting this method if it won't be used in console
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)